man/pcair.Rd

```diff
@@ -22,7 +22,7 @@
 \arguments{
   \item{gdsobj}{An object providing a connection to a GDS file.}
   \item{kinobj}{A symmetric matrix of pairwise kinship coefficients for every pair of individuals in the sample: upper and lower triangles must both be filled; diagonals should be self-kinship or set to a non-missing constant value. This matrix is used for partitioning the sample into the 'unrelated' and 'related' subsets.  See 'Details' for how this interacts with \code{kin.thresh} and \code{unrel.set}.  IDs for each individual must be set as the column names of the matrix. This matrix may also be provided as a GDS object; see 'Details'.}
-  \item{divobj}{A symmetric matrix of pairwise ancestry divergence measures for every pair of individuals in the sample: upper and lower triangles must both be filled; diagonals should be set to a non-missing constant value. This matrix is used for partitioning the sample into the 'unrelated' and 'related' subsets.  See 'Details' for how this interacts with \code{div.thresh}.  IDs for each individual must be set as the column names of the matrix. This matrix may be identical to \code{kinobj}. This matrix may also be provided as a GDS object; see 'Details'.}
+  \item{divobj}{A symmetric matrix of pairwise ancestry divergence measures for every pair of individuals in the sample: upper and lower triangles must both be filled; diagonals should be set to a non-missing constant value. This matrix is used for partitioning the sample into the 'unrelated' and 'related' subsets.  See 'Details' for how this interacts with \code{div.thresh}.  IDs for each individual must be set as the column names of the matrix. This matrix may be identical to \code{kinobj}, or \code{NULL} to ignore ancestry divergence. This matrix may also be provided as a GDS object; see 'Details'.}
```

Comment:
It sort of sounds like making this matrix identical to kinobj to ignore ancestry divergence. 
Maybe we should say:

This matrix may be identical to \code{kinobj}. This matrix may be \code{NULL} to ignore ancestry divergence.

Reply:
Agreed.